scripts/deploy/runner.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [[ -z "$1" ]] || [[ -z "$2" ]]; then
+  echo "PROJECT_ID and SUBMISSION_BUCKET_NAME are mandatory"
+  exit 1
+fi
+
+PROJECT_ID=$1
+SUBMISSION_BUCKET_NAME=$2
+IMAGE_TAG="${3:-latest}"
+
+kubectl create secret generic keys \
```

Comment:
Im not sure about these and where they should live.
also this currently wont work as the `docker-keys.yml` are not in this folder?

Reply:
Not too sure, I was thinking of splitting into a separate script, and we still need to think about how we would pass key files in for preprod/prod.

As for the `docker-keys.yml`, it's relative to where you execute from.
So if you are in `eq-survey-runner` folder, and `docker-keys.yml` is in that folder, it works fine.
If I `cd scripts`, which is 1 folder in, then it would need to be `../docker-keys.yml`.